config.toml.example

```diff
@@ -37,6 +37,11 @@
 		APIKey = "AIzaSyCzI5Gvpq1hPlfjvB81EVc0vLv_11eW7NI"
 		playlistID = "PLsxUDJzU4VKDudlmlYb-ImFKreFKAfFln"
 
+	[pageContext.gmaps]
+		APIKey = "AIzaSyACeeyIgdIhiSxV01OIPiA3MyJDqrOspLQ"
```

Comment:
Erm, isn't the .example supposed to be empty and the regular one not supposed to be commited to the github?

Reply:
It gets onto the public page anyway, no big deal.

Reply:
yeah that is supposed to be the point I think but at the moment to make it to deploy it just uses the example one